app/models/organization.rb

```diff
@@ -110,12 +110,6 @@ def enough_credits?(num_credits_needed)
     credits.unspent.size >= num_credits_needed
   end
 
-  def suspended?
```

Comment:
Are we sure this won't cause problems? The reason Jacob added this was that we use users and orgs interchangeably in same places and previously at least that led to problems when orgs didn't respond to `suspended?`.

Reply:
I am reasonably sure, but I wouldn't be offended if you gut checked this.

I discovered this when I went to add the score check on the user show page and realized that both user and org shared the same `meta` partial.

So I moved the functionality in general outside of the partial and just into the user file (but still within the `:page_meta` block)...

My understanding is that this change made `suspended?` no longer needed. I believe the only reason for this method was the placement of that functionality within the partial file. I hope that between my examination and the regression tests that this hack can indeed be removed. Just trying to leave the code cleaner than I found it. 🏕😅

Reply:
Right, this seems to make intuitive sense, so I trust you and our spec suite on this. Thanks for the context!